contracts/modules/tokenswap/TokenSwapModule.sol

```diff
@@ -24,6 +24,12 @@ contract TokenSwapModule is ModuleBaseWithFee {
     mapping(uint32 => TokenSwap) public tokenSwaps;
     /// Metadata => deal ID
     mapping(bytes32 => uint32) public metadataToDealId;
+    /// Maximum DAOplomat reward (5%)
+    // solhint-disable-next-line var-name-mixedcase
+    uint256 public immutable MAX_REWARD = 500;
+    /// Minimum DAOplomat reward (0.001%)
+    // solhint-disable-next-line var-name-mixedcase
+    uint256 public immutable MIN_REWARD = 1;
```

Comment:
I probably haven't followed the discussion enough, but should we not allow that the reward is not being used for *some* of the deals?

Reply:
Yes so that is a question I have asked last week as well. Because the information in the document was a bit conflicting. This still has to be adapted. What would be the best way to name the varible because it's basically minimum percentage reward if DAOplomat reward is enabled

Reply:
So to recap, you mean add a boolean member to the struct that reflects if a reward is set or not and check if it's set by checking `> 0` instead compare it against a constant?

Reply:
Also, I'm curious if there isn't a way to parse 0.00001 as reward, in which case it would be bigger than `0`, but invalid

Reply:
Hm, on second thought it probably makes no sense to have both an `int` value and a `boolean` - it's a bit of an overkill. Do you mean 0.00001 as in 10000000000000 Wei? And you mean that would be invalid because it's less than the `MIN` value? Not sure if I really understood you right :D

Reply:
Yes that's what I meant (although not expressed as well as you did :D). Could that not give an issue maybe?